docs/curation-review-tut.md

```diff
@@ -0,0 +1,570 @@
+
+
+**Note:** at least one of the keys in `layer_descriptions` must return the value 'raw'
+
+<br/>
+
+#### `obs` section
+```
+obs:
+    tissue_ontology_term_id: UBERON:0000006              #UBERON tissue term
+    assay_ontology_term_id: EFO:0010961                  #EFO assay term
+    disease_ontology_term_id: PATO:0000461               #MONDO disease term or PATO:0000461
+    sex: male                                            #male, female, mixed, unknown, or other
+    ethnicity_ontology_term_id: na                       #HANCESTRO term
+    development_stage_ontology_term_id: HsapDv:0000160   #HsapDv term
+    cell_type_ontology_term_id:
+      cell_label:                                        #this is column name in your obs dataframe (the field that specifies cell type)
+        acinar: CL:0002064                               #mapping between cell types (specified in AnnData object) and cl ontology id
+        alpha: CL:0000171
+        delta: CL:0000173
+        endothelial: CL:0000115
+        epsilon: CL:0005019
+
+```
+
+
+<br/>
+
+In the config file snippet above, the 0 level indentation specifies that we are modifying `obs`. At the first level of indentation, we start adding schema fields to `obs`. If the first level indentation field is followed by an ontology term id (i.e. `tissue_ontology_term_id: UBERON:0000006`), then a column for that field will be created in the obs dataframe (i.e. a column called `tissue_ontology_term_id` will be created), and all of its values will take on the value specified after the colon (i.e. `UBERON:0000006`). Additionally, the cellxgene curation tools will search the ontology term id and create a corresponding column with the appropriately corresponding term (i.e. a column called `tissue` will be created and all values of that column will be `islet of Langerhans`).
+
+On other hand, if the schema field is followed by lines which have further indentation, then the next line in the config file will specify a column whose values should be mapped to ontology terms (with these mappings specified by the subsequent lines). This is typically more relevant for specifiying cell labels. In our example above, two new columns will be added to the `obs` data frame `cell_type_ontology_term_id` and `cell_type`. The original column specifiying cell identity will be retained (and will be tagged with `_original`). The cellxgene curation tools never remove fields from the original dataset, they only add schema relevant information.
```

Comment:
_specifiying -> specifying_ (two instances)

Reply:
thank you, updated